srcpkgs/xst/template

```diff
@@ -14,6 +14,11 @@ homepage="https://github.com/neeasade/xst"
 distfiles="https://github.com/neeasade/xst/archive/v${version}.tar.gz"
 checksum=5cea5a8bae643ff7df7f8322958065edece354c272c0f987b1dcadcdec689f9a
 
+post_extract() {
```

Comment:
No need for `post_extract`. Put the vsed command in `do_install` before `vdoc doc/Xresources`